Cookbook/Technical-Documents/How to use Slack?.md

```diff
@@ -0,0 +1,146 @@
+
+4. In the `Do Not Disturb` you can set the time interval you don't want to receive any Slack notification. Ideally this should enabled during your non-working hours.
+
+On the day to day basis, if you want some quiet time, you can mute notifications temporarly by selecting the notifications icon next to your Slack organization name followed by selecting one of the options.
+
+<img src="Assets/onboarding/slack-notification-muted.png">
+
+### Leave the channel
+
+When you leave a channel you can no longer open it unless you join it again. Also, when someone shares a thread from that channel with you, you will have to join the channel to be able to read it.
+When the channel is private, if you leave it you will have to be invited by one of its members to join it again.
+
+You should leave a channel when:
+
+ - You haven't open it in the last month.
+ - The subject is no longer relevant or obsolete.
+
+### Mute the channel
+
+Muting a channel means you will have access to it but you won't be notified if you are mentioned individually or with `@channel` or `@here`.
+
+You should mute a channel when:
+
+ - The subject is not your main focus but you like to keep yourself updated on what is happening.
+ - It is a private channel and you don't want to lose access to it, but you don't check it on a day to day basis.
+
+
+## How to communicate my schedule?
+
+Your status on Slack should be a way to communicate what is your current status. This can be used to indicate you are working remotely, you are in a meeting or any other status.
+
+### Connect Slack with Outlook
+
+You can integrate your Slack account with your Outlook account. This will give permission for Slack to:
+- notify you of your calendar events
+- change your status to `In Meeting` when you have meetings scheduled in your calendar
+
+To connect the Outlook Calendar app:
+
+- Go to Apps in Slack
+- Search for Outlook Calendar
+- Open a conversation with Outlook Calendar
+- Connect your account
+- Configure your preferences
+
+### Not available
+
+When we are not available on Slack it is advised to change your Slack status so the person who wants to reach you knows why you are unnavailable.
+You can configure your own status however, for some common situations it is better to use the pre-defined ones:
+
+- [Holidays](/OutOffOfficeRequest.md#holidays-) 🌴
+- [Working from home](https://github.com/babylonhealth/ios-playbook/blob/catarina/CNSMR-3278/Cookbook/Technical-Documents/OutOffOfficeRequest.md#working-from-home-) 🏡
+- Off sick 🤒
+- Focused 💭 - A few of us turn-off notifications or turn-off Slack when we want time to focus on a certain task.
+
+Scheduling time slots to focus on a task is encouraged because it helps our productivity. When booking these please be mindful of when and for how long you are unnavailable. It is always better to schedule these slots when you have no meetings in your calendar and are not involved in conversations that are active at that time.
```

Comment:
```suggestion
Scheduling time slots to focus on a task is encouraged because it helps our productivity. When booking these please be mindful of when and how long you are unavailable for. It is always better to schedule these slots when you have no meetings in your calendar and not involved in conversations that are active at that time.
```